about-impl-days.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: "What is `impl Days`"
+permalink: /about_impl_days/
+---
+
+<div class="backdrop" style="background: url(/assets/rome/the-city.jpg) 50% 40%; padding: 0; margin-bottom: 1em;">
+  <div class="popout">
+    <section>
+      <h1>impl Days</h1>
+    </section>
+  </div>
+</div>
+
+{% comment %}
+<section markdown="1">
+## Todays Info
+{% include_relative impl_days_room_info.html %}
+</section>
+{% endcomment %}
+
+<section markdown="1">
+## about:impl_days
+
+{% comment %}
+TODO: add sponsors
+{% endcomment %}
+
+`impl Days` is the sibling event of RustFest&lt;Rome&gt;. `impl Days` is an open-ended time
```

Comment:
It's not open-ended. It's a full day event,  but we tend to get people out the door around 17:00 or so.

Reply:
righty right